ThemeDecorator/ThemeDecorator.js

```diff
@@ -54,6 +54,11 @@ const defaultConfig = /** @lends sandstone/ThemeDecorator.ThemeDecorator.default
 	 */
 	disableFullscreen: false,
 
+	/**
+	 * Font scale for large text mode.
+	 */
+	fontScale: 1,
+
```

Comment:
Please remove from ThemeDecorator config. I think we should handle fontScale value with a prop as textSize.

Reply:
Ok. I removed in ThemeDecorator's config. It seems that this is only needed in ResolutionDecorator.

Reply:
Ok. I removed in ThemeDecorator's config. It seems that this is only needed in ResolutionDecorator.

Reply:
It seems that it still remain in ThemeDecorator`s config. Please check it.

Reply:
totally removed